mkt/lookup/templates/lookup/app_summary.html

```diff
@@ -108,12 +108,6 @@
             {% endfor %}
           </ul>
         </dd>
-        <dt>{{ _('Downloads') }}</dt>
-        <dd class="pretty-list">
-          {% for key, heading in summaries %}
-            <div>{{ heading }}: {{ downloads[key] }}</div>
-          {% endfor %}
-        </dd>
```

Comment:
@eviljeff Updated to just remove this section in the lookup tool. We have a link to statistics on this page already and that will be more accurate than this ever was.